ChangeLog

```diff
@@ -1,3 +1,11 @@
+2021-01-19  Theodora Lo <tlo@bcgsc.ca>
+
+	* Release version 1.2.0
+	* Added `--arcs-long` and `--arks-long` modes to `arcs-make` so
+	users can run the ARCS+LINKS pipeline using long reads as input
```

Comment:
```suggestion
	* Users can now also use long reads to scaffold draft assemblies
	* Added `--arcs-long` and `--arks-long` modes to `arcs-make` so
	users can run the ARCS+LINKS pipeline using long reads as input
```
Just thinking would be good to have a general statement like that so the new feature is super obvious!